en/docs/developer-guides/workspace-management/create-integration.md

```diff
@@ -0,0 +1,45 @@
+# Create an Integration
+
+## Creating an Integration Without Workspace
+When you need to build a simple, standalone integration, you can create it directly without setting up a workspace. This streamlined approach is ideal for independent services.
+
+**Steps:**
+
+1\. Launch WSO2 Integrator: BI and navigate to the integrations view.
+
+2\. Click the **Create New Integration** button.
+
+3\. Fill in **Create Your Integration** details and click **Create Integration**:
+
+- Provide the integration name
+- Provide a package name 
+- Select the path 
+- Provide the organization name and version in **“Optional Configurations”** (Optional)
+
+Now you will see the BI home page.
+
+<a href="{{base_path}}/assets/img/developer-guides/workspace/create-integration-hq.gif"><img src="../../../assets/img/developer-guides/workspace/create-integration-hq.gif" alt="Create Integration Without Workspace" width="80%"></a>
+
+
+## Creating an Integration With Workspace
+Creating an integration within a workspace provides better organization and enables advanced features for managing multiple related integrations.
+
+**Steps:**
+
+1\. Launch WSO2 Integrator: BI and navigate to the integrations view
+
+2\. Click on the **“Create New Integration”** button, add details:
+
+- Provide the integration name 
+- Provide a package name 
+- Select the path
+
+3\. Expand **“Optional Configurations”** and enable  **“Create as workspace”** option
+
+- Provide relevant workspace name
+- Provide the organization name and version (Optional):
+
+  <a href="{{base_path}}/assets/img/developer-guides/workspace/expand_workspace.png"><img src="../../../assets/img/developer-guides/workspace/expand_workspace.png" alt="Expand Workspace Details Form" width="80%"></a>
```

Comment:
can we redo the gifs using proper names of integrations out of a real example? using `sample01`, `sample02` etc are not helping out to understand the real usage.

Reply:
Update screenshots and gif https://github.com/wso2/docs-bi/pull/197/commits/7aec3912da0898be7556a39525db4463a2a5273e